src/api/package/util.ts

```diff
@@ -1,7 +1,7 @@
 export function getAssetsPath(name: string, version: string, scope?: string) {
-  const prefix = `/packages/`;
+  const prefix = `/data/`;
```

Comment:
https://github.com/cdklabs/construct-hub/blob/main/src/backend/shared/constants.lambda-shared.ts#L12

Reply:
Can we somehow reuse these constants across the packages?